### PR TITLE
JupyterLab Updates

### LINKED ISSUE #1248: Impossible to close/move `notebook tutorial` in lab
Observed on 7174d9d

To reproduce:
1. Select "notebook tutorial" from the command palette.
2. Verify that the tab with the tutorial can only be resized but not closed or moved around.

Desired outcome:
It should be possible to close the notebook tutorial.

### PR DESCRIPTION
**Pull request by blink1073**
- Fixes usability issues addressed in other jupyter-js-* repos, and fixes #1248.

- Allows JupyterLab to update to the latest patch version of `jupyter-js-plugins` when building.

- Needed to move the static `build` files to the `notebook/lab` directory so they could be served.

### CODE REVIEW

**Comment by blink1073** [comment]
@minrk, this is good to go and has a lot of utility.

File: notebook/lab/handlers.py [hunk 1]
```diff
@@ -18,12 +19,12 @@ def get(self):
...
 
-
 #-----------------------------------------------------------------------------
 # URL to handler mappings
 #-----------------------------------------------------------------------------
 
-
 default_handlers = [
     (r"/lab", LabHandler),
+    (r"/lab/(.*)", AuthenticatedFileHandler,
```
Comment (minrk) [inline notebook/lab/handlers.py]: I suspect this wants to be a regular StaticFileHandler (or more precisely FileFindHandler, to get identical caching behavior to the existing static files). We don't normally authenticate access to the basic js/css.

Comment (blink1073) [inline notebook/lab/handlers.py]: Thanks, updated.

**Comment by blink1073** [comment]
:party:

**Pull request merged**